testing/visualization_test.py

```diff
@@ -0,0 +1,38 @@
+import unittest, uuid, json
+import os
+from overcooked_ai_py.visualization.extract_events import extract_events
+from overcooked_ai_py.visualization.visualization_utils import create_chart_html
+from overcooked_ai_py.agents.benchmarking import AgentEvaluator
+from overcooked_ai_py.utils import load_from_json, NumpyArrayEncoder
+from utils import TESTING_DATA_DIR
+from overcooked_ai_py.mdp.overcooked_mdp import Recipe
+
+def jsonify(obj):
+    return json.loads(json.dumps(obj, cls=NumpyArrayEncoder))
+
+class TestVisualizations(unittest.TestCase):
```

Comment:
Make sure you add this test to the `/.github/workflows/pythontests.yml` file so it executes on PRs and pushes. While in the future, any file matching `/testing/*_test.py` will get automatically executed, that's not currently the case!

Reply:
done